.github/workflows/test.yml

```diff
@@ -26,6 +26,6 @@ jobs:
       - name: Run tests
         # Runs a script that creates a PostgreSQL client,
         # populates the client with data, and retrieves data
-        run: go test ./...
+        run: go test -coverprofile=coverage.out ./...
```

Comment:
How can this output be viewed?

Reply:
Right now, the output file is temporary (deleted on container destruction) and the only visible output is in the logs, showing a percentage per package that has tests in it (mainly just the `lnmux` and `persistence` packages. I would like to get this reporting to something with easy-to-read output, perhaps CodeCov or shields.io?

Reply:
71%, not bad. It would be nice to view the coverage somewhere, but for me personally it isn't the highest priority because vscode also shows coverage when running tests locally.

Reply:
I've changed this to only output to logs, not to a file. No difference in the end, but then there's no question about what happens with the output: it's only visible in the logs, in case it's wanted.